chronosphere/resource_notification_policy_independent.go

```diff
@@ -142,13 +144,33 @@ func notificationRoutesFromModel(
 			Severity:       sev,
 			Notifiers:      sliceutil.Map(f.NotifierSlugs, tfid.Slug),
 			RepeatInterval: durationFromSecs(f.RepeatIntervalSecs),
+			GroupBy:        notificationRouteFromGroupByModel(f),
 		})
 	}
 	load(warn, m.Warn)
 	load(critical, m.Critical)
 	return out
 }
 
+func notificationRouteToGroupByModel(r intschema.NotificationRoute) *models.NotificationPolicyRoutesGroupBy {
+	if r.GroupBy == nil {
+		return nil
+	}
+	return &models.NotificationPolicyRoutesGroupBy{
+		LabelNames: r.GroupBy,
+	}
+}
+
+func notificationRouteFromGroupByModel(f *models.RoutesNotifierList) []string {
+	if f == nil {
+		return nil
+	}
+	if f.GroupBy != nil {
+		return f.GroupBy.LabelNames
+	}
+	return []string{}
```

Comment:
nil is a valid empty slice, so no need for `[]string{}`,
```suggestion
	if f == nil || f.GroupBy == nil {
		return nil
	}
	return f.GroupBy.LabelNames
```

Reply:
thanks!